tests/unit/DocumentDelivery.spec.ts

```diff
@@ -67,6 +67,23 @@ for (const mock of documentDeliveryCases) {
       expect(wrapper.find('#contact-value').text()).toBe(mock.businessContact.email)
     })
 
+    it('displays additionalLabel labels', () => {
+      wrapper = shallowWrapperFactory(DocumentDelivery, {
+        additionalLabel: 'partners',
+        contactLabel: 'Registered Office'
+      })
+
+      expect(wrapper.findAll('label').at(2).text()).toBe('partners')
+    })
```

Comment:
This test should either be standalone (not in this "for" loop), or should use loop parameters to test different things for the items in `documentDeliveryCases[]`.

Also, please add a blank line between test methods.